packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -139,10 +138,7 @@ export default function BookmarkFeedLayout({
         </Typography>
       </FeedPageHeader>
       <CustomFeedHeader
-        className={classNames(
-          'mb-6',
```

Comment:
The component `CustomFeedHeader` already has the `mb-6`. No need for us to add it again.